vic/vic_run/include/vic_run.h

```diff
@@ -144,13 +144,19 @@ double ErrorIcePackEnergyBalance(double Tsurf, ...);
 double ErrorPrintIcePackEnergyBalance(double, va_list);
 int ErrorPrintSnowPackEnergyBalance(double, va_list);
 int ErrorSnowPackEnergyBalance(double Tsurf, ...);
-int estimate_layer_ice_content(layer_data_struct *, double *, double *,
-                               double *, double *, double *, double *, double *,
-                               double, size_t, size_t, char);
-int estimate_layer_ice_content_quick_flux(layer_data_struct *, double *, double,
-                                          double, double, double, double *,
-                                          double *, double *, double *, double,
-                                          char);
+void estimate_frost_temperature_and_depth(double tmpT[MAX_LAYERS][MAX_NODES][MAX_FROST_AREAS + 1], double tmpZ[MAX_LAYERS][MAX_NODES], double *, double *, double *, double *, double, size_t,
```

Comment:
Don't use the array bounds in these function definitions. Simple say `double ***tmpT`, same for all the others. 

I do like having the variable names in there, but that is not really the convention in the rest of the file, so remove all the variable names. Same for below.